bookmarks/templates/bookmarks/bookmark_list.html

```diff
@@ -14,6 +14,11 @@
             {{ bookmark.resolved_title }}
           </a>
         </div>
+        {% if request.user.profile.display_url %}
+        <div class="url-path truncate">
+          <span class="url-display text-sm">{{ bookmark.url }}</span>
```

Comment:
This should use an anchor tag so you can actually click on it. Can reuse the one from the title, minus the unread / italic styles.

Reply:
Done